src/lib/AppBar.svelte

```diff
@@ -1,135 +1,138 @@
 <script lang="ts">
-  import csmaLogo from '$lib/assets/turtle-logo.png';
+	import csmaLogo from '$lib/assets/turtle-logo.png';
 
-  var isMenuopen = false;
+	var isMenuopen = false;
 </script>
 
-<nav class="app-bar">
-  <a class="branding" href="/">
-      <img id="csmaLogo" src={csmaLogo} alt="The CSMA logo" />
-      <p>CSMA</p>
-  </a>
-  <ul on:click={() => { isMenuopen = false }} on:keypress={() => { isMenuopen = false }} class={`nav-links ${isMenuopen ? '' : 'hide-menu'}`}>
-    <li>
-      <a href='/projects'>Projects</a>
-    </li>
-    <li>
-      <a href='/about'>About</a>
-    </li>
-    <li>
-      <a href='/services'>Services</a>
-    </li>
-    <li>
-      <a href='/contact'>Contact</a>
-    </li>
-  </ul>
-  <button on:click={() => { isMenuopen = !isMenuopen }} class="hamburger-button">
-    {#if isMenuopen}
-      <span class="material-symbols-outlined">
-        close
-      </span>
-    {:else}
-      <span class="material-symbols-outlined">
-        menu
-      </span>
-    {/if}
-  </button>
-</nav>
+<div class="fixed top-0 left-0 w-full bg-none z-10">
+	<nav class="app-bar bg-surface-500 rounded-3xl mx-auto w-fit m-1 border-tertiary-500 border-2">
+		<a class="branding ml-2 mr-2 md:mr-8" href="/">
+			<img id="csmaLogo" src={csmaLogo} alt="The CSMA logo" />
+			<p class="font-heading-token text-2xl ml-1 max-md:hidden">CSMA</p>
+		</a>
+		<!-- svelte-ignore a11y-no-noninteractive-element-interactions -->
```

Comment:
Does our island work with keyboard only users? is this just a linting thing

Reply:
It should since we have it with a keypress handler as well, but would be good to test. That was already like that though so at worst it is not a regression